fr/user/articles/README.md

```diff
@@ -1,11 +1,11 @@
 Articles
 ========
-Articles are the core feature of wallabag. wallabag's content parser, named Graby gets the content from articles, and only the content.
+Les articles sont une fonctionalités clé dans wallabag. Graby, le *parser* de contenus de wallabag récupère les contenus des articles, et uniquement le contenu.
```

Comment:
I would prefer "représentent la caractéristique principale de". Also "parser" could be translated into its french counterpart "parseur" or "extractreur"

Reply:
J'ai mis « sont un concept clé de wallabag »